src/cuitensor.jl

```diff
@@ -79,3 +79,14 @@ function CuMatrix(T::ITensor,i1::Index,i2::Index)
   return CuArray(T,i1,i2)
 end
 
+function ITensors.can_combine_contract(A::ITensor, B::ITensor)
```

Comment:
Maybe better to make this an `NDTensors` function acting on `Tensor` types...

Reply:
That's a change that will have to be made in ITensors, no?

Reply:
Yup.